valhalla/mjolnir/osmway.h

```diff
@@ -6,13 +6,15 @@
 #include <string>
 #include <vector>
 
-#include "mjolnir/osmpronunciation.h"
```

Comment:
couldnt the fix have been as simple as <valhalla/mjolnir/osmpronunciation.h> here, and then don't do any of the other changes?

Reply:
unluckily No, that was the first thing I did LOL

Reply:
oops, that idea seems should work.. I have done something wrong certainly...